pkg/cloud/network.go

```diff
@@ -31,6 +31,7 @@ const (
 	K8sDefaultAPIPort   = 6443
 	NetworkTypeIsolated = "Isolated"
 	NetworkTypeShared   = "Shared"
+	tcp                 = "tcp"
```

Comment:
```suggestion
	NetworkProtocolTCP                 = "tcp"
```
To be consistent with other constants.

Reply:
Good idea.  Thanks!